### PR TITLE
These are the changes @lordmatt made to stop fatal errors on my site

### PR DESCRIPTION
**Pull request by lordmatt**
It seems there are edge cases where some static functions in class-linkbacks-mf2-handler.php return null after failing all the conditionals. Other methods call these but fail to test for null. This leads to some instances of fatal errors. This push works for my site and stops the errors. It may be that better defaults can be found and/or more helpful actions in case of a wrong var type. 

This is in use at openmentions.com.

### CODE REVIEW

File: includes/class-linkbacks-mf2-handler.php [hunk 1]
```diff
@@ -565,7 +568,7 @@ public static function get_representative_author( $mf_array, $source ) {
 			return $mf_array['rels']['author'];
 		}
 
-		return null;
+		return 'someone';
```
Comment (pfefferle) [inline includes/class-linkbacks-mf2-handler.php:571]: If we add this, every following check will break and it will add 'someone' most of the time!

**Review by pfefferle — CHANGES_REQUESTED** [review]
